src/controllers/PermissionsController.js

```diff
@@ -0,0 +1,67 @@
+import models from '../models';
+import response from '../utils/Response';
+
+const { serverResponse } = response;
+
+const { roles } = models;
+
+
+/**
+ *
+ *
```

Comment:
Why are there empty jsdoc lines?

Reply:
I have taken out the empty line. Kindly review @mthamayor